assets/src/edit-story/components/canvas/karma/selection.karma.js

```diff
@@ -87,7 +87,31 @@ describe('Selection integration', () => {
       up(),
     ]);
     expect(await getSelection()).toEqual([element1.id]);
-    await fixture.snapshot();
+    await fixture.snapshot('selection and page menu');
+  });
+
+  it('should show the selection on top of page navigation arrows', async () => {
+    await fixture.events.click(
+      fixture.screen.getByRole('button', { name: 'Add New Page' })
+    );
+    await fixture.events.click(
+      fixture.screen.getByRole('button', { name: 'Previous Page' })
+    );
+    const frame1 = fixture.editor.canvas.framesLayer.frame(element1.id).node;
+    await fixture.events.click(frame1);
+    const fbcr = frame1.getBoundingClientRect();
+    await fixture.events.mouse.seq(({ moveRel, moveBy, down, up }) => [
+      moveRel(frame1, 10, 10),
+      down(),
+      moveBy(
+        fullbleed.getBoundingClientRect().width - 20,
```

Comment:
Note -- still need to fix this to position it correctly based on the position of the nav arrow.